build.xml

```diff
@@ -1027,6 +1028,22 @@
     </copy>
   </target>
 
+  <macrodef name="checktestnameshelper">
+    <sequential>
+      <taskdef name="test-name-check_" classname="org.apache.cassandra.anttasks.TestNameCheckTask" classpath="${test.classes}">
+        <classpath>
+          <path refid="cassandra.classpath.test"/>
+          <path location="${fqltool.build.classes}"/>
+        </classpath>
+      </taskdef>
+      <test-name-check_/>
+    </sequential>
+  </macrodef>
+
+  <target name="check-test-names" depends="_main-jar,stress-build,fqltool-build,resolver-dist-lib,simulator-jars,_build-test">
+    <checktestnameshelper/>
+  </target>
```

Comment:
nit: is there _actually_ need for a separate target?

Reply:
I think so. Why not? There is target for checkstyle as well.

Reply:
YAGNI

Reply:
We may remove that. I dont object. As long as it is called as part of the build, the separate target is not the absolute must.

Reply:
I removed it.